docs/source/guides/staking_guide.rst

```diff
@@ -237,6 +253,35 @@ the address to checksum format in geth console:
 After this step, you're finished with the Staker, and you can proceed to :ref:`ursula-config-guide`.
 
 
+Manage automatic reward restaking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As your Ursula performs work, you can optionally enable the automatic addition of
+all rewards to your existing stake to optimize earnings.  By default this feature is disabled,
+to enable it run:
```

Comment:
I think we talked about leaving re-stake on by default, since that would be the most profitable and common scenario for stakers. Also, use "re-stake" with hyphen in the section title and in the next lines.

Reply:
Yeah right

Reply:
Yeah - I agree  that it makes sense to have restaking enabled by default - just documenting the current behavior.  Is this something we want to control at the contract level?  If not we can add a re-stake enable transaction to the `nucypher stake create` command.

Reply:
We can just leave this as an open issue, because it does make sense to do it at the contract level. What do you think @szotov ?

Reply:
unfortunately no such thing like default value in contract, because all values are zero in the beginning and only can be set in functions. So we can set value it in first deposit if we want, but I'm not sure is it good decision to do it implicitly in deposit function

Reply:
The value can be set when initializing the contract though?

Reply:
Not really, because it's individual structure for each staker
I can do a trick - use something like `notReStake` field. But also issue here: to keep upgradability this should be a new field and old `reStake` wil be as a garbage

Reply:
Or, perhaps, we can have it explicitly set as a required parameter when we start staking (and then the default can be set in Python)